SharkCage/CageChooser/CageChooser.cs

```diff
@@ -18,8 +18,13 @@ static void Main()
 #if DEBUG && FALSE
             var p = new System.Diagnostics.Process();
             p.StartInfo.FileName = "Powershell.exe";
-            p.StartInfo.Arguments = "-ExecutionPolicy Unrestricted -File ..\\..\\install_service.ps1";
+            p.StartInfo.Verb = "runAs";
+            var rootDir = Directory.GetParent(Directory.GetCurrentDirectory()).Parent;
```

Comment:
@DonatJR if I comment out the `&&` a `using System.IO;` is missing in the file but I think Visual Studio has as default "optimize imports" enabled so if I comment out the `&&` again, the using will be removed.

To comment out `using System.IO;` and `&& FALSE` isn't the best solution but for first we can life with it?
If yes, I would commit that directly into the dev branch.

Reply:
The script won't be necessary once we got the deployment sorted out, it was meant for dev purposes so yeah, I think we can live with that. I have a better solution, wait a minute.

Reply:
Alright, the `using System.IO` statement should not be needed anymore now.

Reply:
Awesome, thanks a lot 👍